assembl/static2/js/app/stories/components/debate/brightMirror/sideComment/sideCommentBox.stories.jsx

```diff
@@ -145,22 +193,36 @@ export const defaultSideCommentBox: SideCommentBoxProps = {
   toggleExtractsBox: Function,
   position: { x: 0, y: 0 },
   setPositionToExtract: Function,
-  clearHighlights: Function
+  clearHighlights: Function,
```

Comment:
Function ?! confusion

Reply:
change Function to storybook action (like above)